node/src/Pos/Security/Workers.hs

```diff
@@ -105,7 +105,8 @@ checkForReceivedBlocksWorkerImpl SendActions {..} = afterDelay $ do
             "by ourselves"
         reportEclipse
     repeatOnInterval delF action = runIfNotShutdown $ do
-        () <- action
+        -- REPORT:ERROR 'reportOrLogE' in block retrieval worker.
+        () <$ action `catchAny` \e -> reportOrLogE "Security worker" e
```

Comment:
Why not `` action `catchAny` reportOrLogE "Security worker"``?

Reply:
Dunno, probably I modified this code many times locally. Anyway, I think it's not essential.